### PR TITLE
Release Notes for 2017.20

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
Release Notes for 2017.20 reviewed in email.

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-126536](https://oktainc.atlassian.net/browse/OKTA-126536)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:
@jakubvul-okta Tech content reviewed via email.

### CODE REVIEW

File: _source/_docs/platform-release-notes/platform-release-notes-2017-19.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
```
Comment (jakubvul-okta) [inline _source/_docs/platform-release-notes/platform-release-notes-2017-19.md]: This filename shouldn't have a hyphen between "notes" and "2017".

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 2]
```diff
@@ -36,22 +36,51 @@ For a full description of the new rate limits, see [API Rate Limit Improvements]
...
+
+#### New Profile Property for Apps
+
+The `profile` property in the Apps API accepts any well-formed JSON schema. You can specify properties in the profile and then access them in API requests.
+For example:
+
+* Add an app manager contact email address.
+* Use the profile to define a whitelist of groups that you can then reference and pass as claims using the [Okta Expression Language function `getFilteredGroups`](/reference/okta_expression_language/index.html#group-functions).
+
+For more information, see the [Apps API](/docs/api/resources/apps.html#set-profile-property-for-openid-connect-apps).
```
Comment (jakubvul-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: This link doesn't exist in this branch, though it does exist in the weekly branch.

File: _source/_docs/platform-release-notes/platform-release-notes-2017-19.md (hunk 1 above)
Comment (mystiberry-okta) [inline _source/_docs/platform-release-notes/platform-release-notes-2017-19.md]: fixed.

File: _source/_docs/platform-release-notes/platform-release-notes.md (hunk 2 above)
Comment (mystiberry-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: I did git rebase -i origin/weekly and checking now to see if that fixed it.

Comment (mystiberry-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: @jakubvul-okta ready for your re-review.

**Pull request merged**